kinto_client/__init__.py

```diff
@@ -261,6 +264,12 @@ def delete_bucket(self, bucket=None, safe=True, last_modified=None):
         resp, _ = self.session.request('delete', endpoint, headers=headers)
         return resp['data']
 
+    def delete_buckets(self, safe=True, last_modified=None):
```

Comment:
Wow. So this deletes all the buckets an user has admin rights on? I wonder if we really need this?